src/Main.elm

```diff
@@ -0,0 +1,51 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Browser.Events
+import Html exposing (Html, button, div, text)
+import Html.Attributes as Attributes exposing (style)
+import Html.Events exposing (onClick)
+import Json.Decode
+import Robot
+
+main =
+    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }
+
+
+type alias Model =
+    { blue : Robot.Model, red : Robot.Model}
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
+    ( { blue = Robot.init 5 5 "blue", red = Robot.init 50 50 "red"}, Cmd.none )
+
+
+type Msg
+    = BlueMsg Robot.Msg
+    | RedMsg Robot.Msg
+    | KeyDown Int
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    Browser.Events.onKeyDown (Json.Decode.map KeyDown Html.Events.keyCode)
```

Comment:
Use `key` instead of `keyCode`. `keyCode` is about to be deprecated, and `key` is nicer to use anyways.